app/uk/gov/hmrc/traderservices/views/StartView.scala.html

```diff
@@ -30,5 +27,49 @@
 
 @govukLayout(pageTitle = Some(messages("view.start.title"))) {
 
+    @html.h1(messages("view.start.heading"))
+
+    @html.h2(messages("view.start.route-one"))
+
+    @html.ul(
+        messages("view.start.route-one.bullets.1"),
+        messages("view.start.route-one.bullets.2"),
+        messages("view.start.route-one.bullets.3")
+    )
+
+    @html.a("/trader-services/route-one", messages("view.start.route-one.link.text"))
+
+    <br><br>
+
+    @html.h2(messages("view.start.reclaim"))
+
+    @html.p{
+        @html.p{
+            @messages("view.start.reclaim.description")
+        }
+        @html.a("/trader-services/reclaim", messages("view.start.reclaim.link.text"))
+    }
```

Comment:
I can't find this page inside the app, so I can't be sure what html code this generates, but these look like nested P tags, which is incorrect html. There are few more instances below.